src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -192,6 +196,42 @@ public async Task UpdateStateAsync(IProgress<IItemLoaderState> progress, Cancell
             NuGetEventTrigger.Instance.TriggerEvent(NuGetEvent.PackageLoadEnd);
         }
 
+        private async Task<SearchResult<IPackageSearchMetadata>> CombineSearchAsync(
+            ContinuationToken continuationToken,
```

Comment:
You are not using the continuation token

Reply:
Sorry, missed this comment earlier. This method has been updated since then and uses the continuation token.